### PR TITLE
add arxiv HTML5(ar5iv) link for firefox

### PR DESCRIPTION
**Pull request by fokx**
Hello, I find it often useful to view arxiv papers in HTML5 instead of PDF. The official instruction is to replace any arXiv article URL by changing the X to a 5.
This PR simply adds the link of HTML5 version next to 'Direct Download' link on the abstract page.
Really hope you can merge it.

### CODE REVIEW

**Comment by fokx** [comment]
[withAr5iv](https://github.com/polymonyrks/withAr5iv) is not available in Firefox and there's no need to install two extensions just for adding simple functionality. Also I think there's no harm adding the ar5iv link alongside direct-download link, so why bother adding additional configurable option?
[arxiv-vanity](https://www.arxiv-vanity.com/) is not official product of arxiv. Authors usually provide arxiv link and it will be really convenient to click a link instead of modifying browser navigation bar link.

**Comment by j3soon** [comment]
I agree with you that the ar5iv feature is useful.

To be specific, my concerns are as follows:

Silently injecting links is quite intrusive. A user once told me that they thought the `Direct Download` link is a feature provided by arXiv.

Since the `Direct Download` feature may speed up the workflow tremendously (and still links to `arxiv.org`), I concluded that its benefits outweigh its downsides and still decided to inject it.

As for other sites that aren't an official product of arXiv, I'm hesitant to put their links directly below the `Direct Download` link. Since it may confuse users to think that ar5iv/arxiv-vanity is a official feature provided by arXiv. Moreover, it is not suitable to put ar5iv in the `Download` section.

I'm thinking of introducing a dedicated section, maybe named as `External Links`, similar to the `References & Citations` section:

image

As for the configurable options, I previously thought that we can let users configures the links in this section. But on second thought, I agree that introducing this option is not necessary.

**TL;DR**: I think we can put the ar5iv link in a dedicated section named as `External Links`. I may add other useful sites (such as arxiv-vanity) in this section in the future.

**Comment by fokx** [comment]
I think your advice is valid.
I have added a commit putting direct-download, ar5iv link and arxiv-vanity link in new section 'Other Links' below 'References & Citations' section.
Also, ar5iv is an official website hosted by arxiv. See [this](https://ar5iv.labs.arxiv.org/) and [this](https://blog.arxiv.org/2022/02/21/arxiv-articles-as-responsive-web-pages/).

**Comment by j3soon** [comment]
Thanks, I'll merge it after pushing some maintenance commits.

Since 2021 (last update of this plugin), the guidelines of uploading to extension/add-ons stores have changed. For example, requiring Manifest v3 (#16). Therefore, I need to fix those issues before adding this feature.

**Comment by j3soon** [comment]
I'll publish this after fixing #21, thanks!

**Comment by j3soon** [comment]
**Update**: This feature is included in v1.7.1 and has been published across all 3 browsers.
